src/views/GovernanceReviewTeam/RequestOverview.tsx

```diff
@@ -153,7 +167,12 @@ const RequestOverview = () => {
             />
             <Route
               path="/governance-review-team/:systemId/business-case"
-              render={() => <BusinessCaseReview businessCase={businessCase} />}
+              render={() => (
+                <BusinessCaseReview
+                  businessCase={businessCase}
+                  grtFeedbacks={grtFeedbacks?.data?.grtFeedbacks}
```

Comment:
Another nit pick: Curious if the team has any suggestions on naming? reading `grtFeedbacks?.data?.grtFeedbacks` seems a bit strange. It reminds me how our system intake reducer reads `systemIntake.systemIntake` and I never liked that.

Reply:
Good catch, let me fix that